test/integration/etl_test.py

```diff
@@ -191,6 +191,6 @@ def test_extract_xbrl(self, ferc1_engine_dbf):
             for table_type, df in xbrl_tables.items():
                 # Some raw xbrl tables are empty
                 if not df.empty and table_type == "duration":
-                    assert (df.report_year >= 2021).all() and (
+                    assert (df.report_year >= 2020).all() and (
```

Comment:
Records which pertain to 2020, but were reported in 2021 will now have a `report_year` of 2020, so make this check more permissive.